### DISCUSSION TITLE
How to set the ADME parameter when you only know: the proportion of the drug prototype and the metabolites metabolized by CYP3A4 in the urine？

### DISCUSSION THREAD

**Discussion by RoachWang** [Open-Systems-Pharmacology/Forum#1585] [How-Tos] [answered]
I don't particularly understand how drug elimination is modeled in PK-SIM.
Right now there is a drug that I only know about, it's total oral clearance, and renal clearance. As well as the proportion of its prototype, and metabolites metabolized by CYP3A4 in the urine.
How do I set the ADME parameters in the “Compound” section?
Similar references would be appreciated.
Thanks for your help!

**Comment by StephanSchaller** [comment, marked as answer]
Here is a link to the Manual for this: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties

**Reply by RoachWang** [reply]
thank you very much
